Documentation/using-dex.md

```diff
@@ -0,0 +1,186 @@
+# Writing apps that use dex
+
+Once you have dex up and running, the next step is to write applications that use dex to drive authentication. Apps that interact with dex generally fall into one of two categories:
+
+1. Apps that request OpenID Connect ID tokens to authenticate users.
+  * Used for authenticating an end user.
```

Comment:
You might need:
```
1. Thing
    * four spaces in
    * something else
2. Even more
    * etc, etc.
```
Formats to:

1. Thing
    * four spaces in and one newline break above/below
    * something else
2. Even more
    * etc, etc.

At least that renders correctly in the PR comments 😐